plugins/woocommerce-blocks/assets/js/templates/revert-button/index.tsx

```diff
@@ -98,21 +98,22 @@ const RevertClassicTemplateButton = () => {
 							} }
 						>
 							{ __(
-								'Revert to Classic Product Template',
+								'Revert to Classic Template',
 								'woocommerce'
 							) }
 						</Button>
 						<span>
 							{ createInterpolateElement(
 								__(
-									`The <strongText /> template doesn’t allow for reordering or customizing blocks, but might work better with your extensions`,
+									`The <strongText /> template doesn’t allow for reordering or customizing blocks, but might work better with your extensions.`,
 									'woocommerce'
 								),
 								{
 									strongText: (
 										<strong>
-											{ template?.record?.title
-												?.rendered ?? '' }
+											{ template?.record?.title?.rendered
+												? `Classic ${ template.record.title.rendered }`
```

Comment:
I suggest we make it aligned with the recent rename (https://github.com/woocommerce/woocommerce/pull/44931), which was:

- WooCommerce Product Attribute Block -> Product Attribute (Classic)
- WooCommerce Product Grid Block -> Product Grid (Classic)
- etc.

```suggestion
												? `${ template.record.title.rendered } (Classic)`
```

Reply:
Makes sense. I have made the changes in 4b7972f. Thanks for the suggestion. I will update the PR description accordingly.